### PR TITLE
Refactor and minor features

### PR DESCRIPTION
**Pull request by floitsch**
Prepares for a v2 release.

### CODE REVIEW

**Comment by floitsch** [comment]
I have ported Artemis to use the new v2 API: https://github.com/toitlang/pkg-cli/pull/63

**Comment by floitsch** [comment]
As discussed: renamed 'info' to 'inform' and 'warning' to 'warn'.

File: README.md [hunk 1]
```diff
@@ -266,18 +270,18 @@ add the following options to the root command:
...
 ``` toit
-import cli
+import cli show Cli
 
-some-chatty-method app/cli.Application:
-  ui := app.ui
+some-chatty-method cli/Cli:
+  ui := cli.ui
   ui.debug "This is a debug message."
   ui.verbose "This is a verbose message."
```
Comment (kasperl) [inline README.md:282]: Maybe not the best verby name.

`verbose`, `interactive`, `result` are all kind of weird now - and got a bit weirder by the info -> inform, warning -> warn cleanup.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: README.md (hunk 1 above)
Comment (floitsch) [reply README.md:282]: I will do a follow-up PR.
As discussed in person:
```
emit --info
emit --warning
...
```

**Pull request merged**